codebreaker.gemspec

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require_relative 'lib/codebreaker/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'codebreaker'
+  spec.version       = Codebreaker::VERSION
+  spec.authors       = ['Dmitry Latyshev']
+  spec.email         = ['gliffy@ukr.net']
+
+  spec.summary       = 'Codebreaker Game'
+  spec.description   = 'Gem for Codebreaker Game'
+  spec.homepage      = 'https://github.com/Dimasfullxq/codebreaker-gem'
+  spec.license       = 'MIT'
+  spec.required_ruby_version = Gem::Requirement.new('>= 2.3.0')
+
+  spec.metadata['allowed_push_host'] = "TODO: Set to 'http://mygemserver.com'"
+
+  spec.metadata['homepage_uri'] = spec.homepage
+  spec.metadata['source_code_uri'] = 'https://github.com/Dimasfullxq/codebreaker-gem'
+  spec.metadata['changelog_uri'] = 'https://github.com/Dimasfullxq/codebreaker-gem/CHANGELOG.md'
+
+  # Specify which files should be added to the gem when it is released.
+  # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
+  spec.files = Dir.chdir(File.expand_path(__dir__)) do
+    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  end
+  spec.bindir        = 'exe'
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ['lib']
+  spec.add_development_dependency 'fasterer'
+  spec.add_development_dependency 'rspec'
+  spec.add_development_dependency 'rubocop'
+  spec.add_development_dependency 'rubocop-performance'
+  spec.add_development_dependency 'rubocop-rspec'
+  spec.add_development_dependency 'simplecov'
+  spec.add_development_dependency 'solargraph'
```

Comment:
nit: please add versions for packages.

Reply:
It's gonna be more convenient. You're right 🚓